discovery-infra/tests/conftest.py

```diff
@@ -49,7 +49,10 @@ def is_qe_env():
                  "service_cidr": utils.get_env('SERVICE_CIDR', '172.30.0.0/16'),
                  "cluster_cidr": utils.get_env('CLUSTER_CIDR', '10.128.0.0/14'),
                  "host_prefix": int(utils.get_env('HOST_PREFIX', '23')),
-                 "iso_image_type": utils.get_env('ISO_IMAGE_TYPE', consts.ImageType.FULL_ISO)}
+                 "iso_image_type": utils.get_env('ISO_IMAGE_TYPE', consts.ImageType.FULL_ISO),
+                 "worker_vcpu": 4,
```

Comment:
It must not be hardcoded values add an option to read them from os env

Reply:
and add 4 to consts

Reply:
fixed